src/sipi.cpp

```diff
@@ -1000,10 +995,6 @@ int main(int argc, char *argv[]) {
 
         try {
             std::cout << std::endl << "Ivan was here" << std::endl;
```

Comment:
Is this line still needed? :P

Reply:
I like it ;-) It is also used in some queries in Infra.

Reply:
Sadly, the only way to figure out if Sipi restarted.